onnx/common/path.h

```diff
@@ -8,17 +8,43 @@
 #pragma once
 
 #include <string>
+#ifdef _WIN32
+#include <windows.h>
+#include <filesystem>
+#include "onnx/checker.h"
+#endif
 
 namespace ONNX_NAMESPACE {
 
 #ifdef _WIN32
-const std::string k_preferred_path_separator = "\\";
+constexpr char* k_preferred_path_separator = "\\";
```

Comment:
You missed a const. 
constexpr const char*

Reply:
And  why it is a string instead of a single char?

Reply:
Thanks for catching this unintended change. Just updated.